app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Outdata vaccine")

```

Comment:
The error message in the `OutdatedVaccineError` exception is not very clear. It would be better to include more details about the error. For example, 'The visitor's vaccine is outdated. All visitors should have a valid vaccine.'